app/models/pending_loan.rb

```diff
@@ -6,8 +6,31 @@ def initialize(pending_loan)
   def projects
     output = {}
     @pending_loan.each do |project_id, loan_amount|
-      output[Project.find(project_id)] = loan_amount
+      output[Project.find(project_id)] = loan_amount.to_i
     end
     output
   end
+
+  def calculate_total_cost
+    projects.values.reduce(:+)
```

Comment:
Why is this "values" here? In the model we have "price" and above loan has an "amount", so maybe we should use the same language for consistency?